<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>497</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>HP Simplified</family>
    <pointsize>20</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Human Posture-Based Game Controller -Setting-</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">

      <!-- row 0, col 0 : Both hands up -->
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="layout_both">
        <item>
         <widget class="QLabel" name="label_both_img">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_both">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string>Both hands up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyEdit_Both_hands_up">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_both_mods">
          <item>
           <widget class="QCheckBox" name="checkShift_Both">
            <property name="text">
             <string>Shift</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkCtrl_Both">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAlt_Both">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>

      <!-- row 0, col 1 : Left hand up -->
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="layout_left">
        <item>
         <widget class="QLabel" name="label_left_img">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_left">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string>Left hand up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyEdit_Left_hand_up">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_left_mods">
          <item>
           <widget class="QCheckBox" name="checkShift_Left">
            <property name="text">
             <string>Shift</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkCtrl_Left">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAlt_Left">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>

      <!-- row 1, col 0 : Right hand up -->
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="layout_right">
        <item>
         <widget class="QLabel" name="label_right_img">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_right">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string>Right hand up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyEdit_Right_hand_up">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_right_mods">
          <item>
           <widget class="QCheckBox" name="checkShift_Right">
            <property name="text">
             <string>Shift</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkCtrl_Right">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAlt_Right">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>

      <!-- row 1, col 1 : Idle -->
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="layout_idle">
        <item>
         <widget class="QLabel" name="label_idle_img">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>150</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_idle">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string>Idle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="keyEdit_Idle">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_idle_mods">
          <item>
           <widget class="QCheckBox" name="checkShift_Idle">
            <property name="text">
             <string>Shift</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkCtrl_Idle">
            <property name="text">
             <string>Ctrl</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAlt_Idle">
            <property name="text">
             <string>Alt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>

      <!-- row 2 : Start / Close buttons -->
      <item row="2" column="0">
       <widget class="QPushButton" name="StartButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="CloseButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>

     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
